src/main/java/com/documaster/rms/noark/ws/serialization/NoarkEnumJsonSerializer.java

```diff
@@ -3,7 +3,6 @@
 import java.io.IOException;
 
 import com.documaster.rms.noark.ws.constants.NoarkEnum;
-import com.documaster.rms.noark.ws.serialization.NoarkDeserializerException;
```

Comment:
I don't think we should say serialize can throw this exception either.

Reply:
Fixed with [cfb6e44](https://github.com/documaster/ws-client-n5-java/pull/34/commits/cfb6e444d6c1e2ca3306e946a50749cd0c00b659)